src/aerie_cli/app.py

```diff
@@ -53,9 +58,40 @@ def set_alternate_configuration(configuration_identifier: str):
 def setup_global_command_context(hasura_admin_secret: str):
     CommandContext.hasura_admin_secret = hasura_admin_secret
 
+def setup_logging(debug: bool):
```

Comment:
Suggest centralizing this logic in `src/utils/logger.py`. Ideally, that module can have a function like `set_log_level` which is the callback for the `--debug` option.